### PR TITLE
V0.7

### PR DESCRIPTION
**Pull request by gerardofn**

### CODE REVIEW

File: plugin/vt_ida/plugin_loader.py [hunk 1]
```diff
@@ -297,85 +380,39 @@ def write_config(self):
...
-        logging.error('[VT Plugin] Unable to connect to VirusTotal.com')
+    try:
+      response = requests.get(url, headers=headers)
+    except:
+      logging.error('[VT Plugin] Unable to check for updates.')
+      return False
 
-      if response.status_code == 404:  # file not found in VirusTotal
+    if response.status_code == 200:  # unable to check version
+      if float(response.text) > float(VT_IDA_PLUGIN_VERSION):
```
[the remark this answers is not in the document]
Comment (gerardofn) [reply plugin/vt_ida/plugin_loader.py]: The content of the VERSION file is float number (represented as a string). The same happens with the variable VT_IDA_PLUGIN_VERSION.

[the remark this answers is not in the document]
Comment (gerardofn) [reply plugin/vt_ida/plugin_loader.py]: I guess you've seen the v0.7beta in the README.md file, it's the only please where this string appears.

**Comment by gerardofn** [comment]
All changes applied

File: plugin/vt_ida/plugin_loader.py [hunk 2]
```diff
@@ -391,6 +390,29 @@ def write_config(self):
       return False
     return True
 
+  @staticmethod
+  def __compare_versions(current, new):
+    current_ver = current.split('.', 1)
+    new_ver = new.split('.', 1)
+
+    if len(current_ver) == 2 and len(new_ver) == 2:
+
```
Comment (winden-g) [inline plugin/vt_ida/plugin_loader.py]: Add this sub-function:

def _normalize(a,b):
      while len(a) > len(b):
        b = '0' + b
      while len(b) > len(a):
        a = '0' + a
    return a,b


then use it multiple times to do the normalization of each field.

**Pull request merged**